appleseed-Installer/Product.wxs

```diff
@@ -0,0 +1,632 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="{EEC9E532-B45A-4B0A-B2AA-1762CFB002D8}" Name="appleseed-MAX" Language="1033" Version="1.0.0.1" Manufacturer="appleseed" UpgradeCode="{46137F8F-2BD6-499C-ADDA-EC812E8E978B}">
+		<Package InstallerVersion="300" Compressed="yes" Platform="x64" InstallScope="perMachine" />
+    <Icon Id="icon.ico" SourceFile="Resources\icon.ico"/>
+    <Property Id="ARPPRODUCTICON" Value="icon.ico" />
+
+
+    <UIRef Id="SetupDialogUI" />
+
+    <Property Id="MAX2017" Value="{}" Secure="yes" />
+    <Property Id="MAX2018" Value="{}" Secure="yes" />
+    <Property Id="MAX2019" Value="{}" Secure="yes" />
+    <Property Id="MAX2020" Value="{}" Secure="yes" />
+    <Property Id="MAX2021" Value="{}" Secure="yes" />
+    
+    <Binary Id="bgImage" SourceFile="Resources/Bg.png" />
+    <Binary Id="cancelbtn" SourceFile="Resources/btCancel.png"/>
+    <Property Id="Cancel">cancelbtn</Property>
+    
+    <Property Id="INSTALLDIR" Value="ROOTDIR" />
+
+		<MajorUpgrade DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+		<MediaTemplate EmbedCab="yes"/>
+
+    <CustomAction Id='DIRCA_TARGETDIR' Property='ROOTDIR' Value='c:\' Execute='firstSequence' />
+    <CustomAction Id='DIRCA_PLUGINDIR' Property='PLUGINDIR' Value='plugins' Execute='firstSequence' />
+    <CustomAction Id='DIRCA_MAPS' Property='MAPS' Value='maps' Execute='firstSequence' />
+    <CustomAction Id='DIRCA_LUXMAPS' Property='LUXMAPS' Value='LuxMap' Execute='firstSequence' />
```

Comment:
There are several mentions of "Lux" in this file, we probably want those to be changed.